README.rst

```diff
@@ -77,7 +82,7 @@ Verify your installation
 
 Check that you can start PyAnsys Math from Python by running this code:
 
-.. code:: python
+.. code:: ipython3
```

Comment:
I'm not sure ``ipython3`` is a supported pygment lexer:

https://pygments.org/languages/

```suggestion
.. code:: python3
```

Reply:
It seems to be accepted locally, but indeed I don't see it in the link you sent. I will make the changes!